tests/unit/packaging/test_tasks.py

```diff
@@ -624,6 +626,8 @@ def find_service(name=None):
                         "maintainer_email": form_factory["maintainer_email"].data
                         or None,
                         "license": form_factory["license"].data or None,
+                        "license_expression": None,
+                        "license_files": [],
```

Comment:
Should these have new entries added in `input_parameters` to feed the `form_factory` and follow the `... or None` pattern? (I haven't read this much yet, so am not certain of the test usage, but it seems like the other fields follow that pattern, so I noted it.)

Reply:
For now no, the test ensure that the correct result comes out for BigQuery when no keys exist in the dist_metadata that comes in.